.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Summary
+
+<!-- Please reference the issue this PR addresses. -->
+Addresses issue #
+
+## Checklist
+
+- [ ] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
+- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
+- [ ] My code follows the [coding standards](https://github.com/ampproject/amp-wp/contributing/engineering.md#coding-standards).
+- [ ] My code has proper inline documentation.
+- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).
```

Comment:
The user will be prompted to sign it if they haven't already. Is this checkbox needed?

Reply:
Removed